midend/parserUnroll.cpp

```diff
@@ -347,7 +347,7 @@ class ParserSymbolicInterpreter {
     /// Returns pointer to genereted statement if execution completes successfully,
     /// and 'nullptr' if an error occurred.
     const IR::StatOrDecl* executeStatement(ParserStateInfo* state, const IR::StatOrDecl* sord,
-                                           ValueMap* valueMap) {
+                          ValueMap* valueMap) {
```

Comment:
I see no reason to change indentation here. All of our code is indented using this style.

Reply:
Ok, I'll revert it back